files/build_templates/qos_config.j2

```diff
@@ -233,6 +233,7 @@
         "global": {
             "dscp_to_tc_map"  : "AZURE"
         }{% if PORT_ACTIVE %},{% endif %}
+
```

Comment:
do we need it?

Reply:
Yes, I found the generated json file will be like below without the empty line.
```
"global": {
            "dscp_to_tc_map"  : "AZURE"
        }, "Ethernet0": {
            "dscp_to_tc_map": "[DSCP_TO_TC_MAP|AZURE]",
            "pfc_enable": "3,4",
            "pfc_to_queue_map": "[MAP_PFC_PRIORITY_TO_QUEUE|AZURE]",
            "pfcwd_sw_enable": "3,4",
            "tc_to_pg_map": "[TC_TO_PRIORITY_GROUP_MAP|AZURE]",
            "tc_to_queue_map": "[TC_TO_QUEUE_MAP|AZURE]"
        },
```
So I added this empty line. 
There is also empty line after other `{% if not loop.last %},{% endif %}`